bokehjs/src/coffee/models/selections/interaction_policy.ts

```diff
@@ -13,6 +14,13 @@ export abstract class SelectionPolicy extends Model {
       return false
     } else {
       source.selected.update(hit_test_result, final, append)
+
+      //new selection created in order for python-side change detection machinery
+      //to detect change in the source's selected property.
```

Comment:
On a very remote side note, as you already used a period, make it a full sentence (start with a capital letter). It will look much more tidier than this. Also use single whitespace between comment indicator and comment content, for consistency with other code.